app/src/components/TokenCard.tsx

```diff
@@ -13,7 +13,7 @@ export const TokenCard = (props: ITokenCardProps): React.ReactElement => {
   return (
     <LinkBase target={`/tokens/${props.token.tokenId}`} isFullWidth={true}>
```

Comment:
LinkBase has props isFullWidth={true} that is why the token displays that way on desktop